api/src/main/java/org/apache/cloudstack/api/command/admin/systemvm/MigrateSystemVMCmd.java

```diff
@@ -126,30 +126,30 @@ public void execute() {
             throw new InvalidParameterValueException("Either hostId or storageId must be specified");
```

Comment:
@weizhouapache do you need to remove this condition else either hostid or storageid must be provided